src/styl/responsive/mobile.styl

```diff
@@ -33,29 +33,59 @@ $transition-speed = 300ms
   opacity: 1
   transition: opacity $transition-speed
 
-  // This class toggles to open/close the video
-  &.video
-    #player-wrapper
-      height: 100vh
-
   // This class toggles to show/hide the video (eg, when buffering)
   &.show
+    #player-wrapper
+      transition: none
     #player-container
       opacity: 1
       transition: opacity $transition-speed
+      height: 100%
+      transition: none
+
+  // This class toggles to open/close the video
+  &.video
+    #player-wrapper
+      width: 100%
+    #player-container
+      height: 100%
+      overflow: hidden
+
+@media (orientation: portrait)
+  #portraits
+    // This class toggles to open/close the video
+    &.video
+      #player-wrapper
+        height: unquote('calc((100vw / 16) * 9)')
+        #player-container
+          width: 100%
+
+$header-height = 89px
+$filters-height = 29px
+
+@media (orientation: landscape)
+  #portraits
+    // This class toggles to open/close the video
+    &.video
+      #player-wrapper
+        height: calc('100vh - ' + ($header-height + $filters-height))
+        #player-container
+          width: calc('((100vh - ' + ($header-height + $filters-height) + ') / 9) * 16')
```

Comment:
This doesnt seem to be rendering correctly. The video crops on the right in mid sizes:

![screen shot 2017-10-10 at 18 32 12](https://user-images.githubusercontent.com/845186/31401073-95e0eaf0-ade9-11e7-8ef8-ca39f371dc77.png)

Would it not be possible to just use the same `56.25vh` height value as per the portrait?